scripts/badger/process_bribes_graviaura.py

```diff
@@ -137,6 +154,33 @@ def sell_weth():
     SAFE.post_safe_tx()
 
 
+def buy_aura(usdc_mantissa):
+    usdc_mantissa = int(usdc_mantissa)
+
+    USDC = interface.ERC20(r.treasury_tokens.USDC, owner=VAULT.account)
+    BADGER = interface.ERC20(r.treasury_tokens.BADGER, owner=VAULT.account)
+    AURA = interface.ERC20(r.treasury_tokens.AURA, owner=VAULT.account)
+    GRAVI_AURA = interface.ITheVault(r.sett_vaults.graviAURA, owner=VAULT.account)
+
+    proc = GreatApeSafe(PROCESSOR.address)
+    proc.take_snapshot([USDC, BADGER, AURA, GRAVI_AURA])
+
```

Comment:
maybe before placing the order is a good measure to check that indeed we have the usdc balance to avoid what happens in the past, at least we could get an alert in the terminal to be aware

Reply:
yea that is a good idea, although that should happen on the `Cow` class level, not here. i will write a ticket.